stats.go

```diff
@@ -369,13 +379,15 @@ func (s *statStore) Flush() {
 	}
 	s.mu.RUnlock()
 
-	s.counters.Range(func(key, v interface{}) bool {
-		// do not flush counters that are set to zero
-		if value := v.(*counter).latch(); value != 0 {
-			s.sink.FlushCounter(key.(string), value)
+	for _, name := range s.counters.Keys() {
```

Comment:
Since we're copying in the LRU code can we add a method that returns a slice of keys and values (this only increases the slices memory footprint by 50%)? Otherwise we have to lock/unlock a mutex and search for the value in the LRU map on each iteration.

Reply:
Yep, seems reasonable.